src/styles/pages/_lte.scss

```diff
@@ -0,0 +1,219 @@
+body.lte {
+  a {
+    cursor: pointer;
+  }
+
+  .hidden {
+    display: none;
+  }
+
+  .lte-title {
```

Comment:
@sjwmoveon do you think this should this be a global `hero-title` or something like that?

Reply:
I think it's fine to leave like this.  We'll work on more generic components with the style guide project.